src/main/kotlin/de/bund/digitalservice/useid/transactioninfo/AdditionalInfoDto.kt

```diff
@@ -0,0 +1,32 @@
+package de.bund.digitalservice.useid.transactioninfo
+
+import org.springframework.data.annotation.CreatedDate
+import org.springframework.data.annotation.Id
+import org.springframework.data.annotation.LastModifiedDate
+import org.springframework.data.relational.core.mapping.Column
+import org.springframework.data.relational.core.mapping.Table
+import java.time.LocalDateTime
+import java.util.UUID
+
+@Table("additional_info")
+data class AdditionalInfoDto(
+    @Column("useid_session_id")
+    val useidSessionId: UUID,
```

Comment:
Die Schreibweise davon variiert stark. Ich glaube, ansonsten wird `useIDSessionId` verwendet..

Reply:
Stimmt, die Schreibweise variiert. Die `IdentificationSession` entity nutzt jedoch die gleiche Schreibweise. Wir sollten uns auf eine Schreibweise einigen, die wir überall verwenden. Was denkst du?

Reply:
Applied a consistent naming with `useIdSessionId` and `eIdSessionId`